src/Console/InfectionContainer.php

```diff
@@ -173,6 +177,10 @@ public function __construct(array $values = [])
 
             return (new MutatorsGenerator($mutatorConfig))->generate();
         };
+
+        $this['metrics'] = function (): MetricsCalculator {
```

Comment:
`Pimple` caches the values the container provides, meaning that it is created on the first run, and then every time it is requested again it returns the same object.